admin-console/src/main/webapp/src/pages/ApplicationPage.tsx

```diff
@@ -77,20 +77,20 @@ export const ApplicationPage: React.FunctionComponent = () => {
         setEventTypes({});
     };
 
-    const handleSubmit = React.useCallback(() => {
+    const handleSubmit = React.useCallback((eventType) => {
```

Comment:
Here we were using the global one, and not the one that was given by the CreateEditModal component. Basically it was never saving the changes.